### ISSUE TITLE
Scrolling is unusably janky on Android 8

### ISSUE THREAD

**Issue by hungryseashell** [AndBible/and-bible#2068]
Scrolling is broken for me.
If I don't lift my finger from the screen (dragging) it kinda works, but with like a very low fps.
If I fling the screen it moves all around the place. Sometimes it just stops in the middle of a flinging scroll, sometimes it moves in the opposite direction actually going a couple of chapters back when scrolling forward.
After a little scrolling around the app freezes completely.

The last version scrolling was working perfectly smooth was on beta-607.1 and on the next version (production-608) it is broken.

Maybe this is related: https://github.com/AndBible/and-bible/issues/1765

My Android version is 8.0.0
Webview is Chrome 97.0.4692.98

**Comment by tuomas2** [comment]
Agreed it is related to #1765 and  #1756 and we think it is bug in Google's webview, and we have reported it to them: https://bugs.chromium.org/p/chromium/issues/detail?id=1265269.

**Comment by tuomas2** [comment]
Hmm got Android 8 phone now (Honor 8 lite), tested and feels usable. Testing with FinRK module.

**Comment by hungryseashell** [comment]
Well it's not usable for me when I flick for long scroll and it scrolls in the opposite direction.

**Comment by tuomas2** [comment]
What kind of setup did you get this from? Could you provide screen capture video to illustrate how it looks like?
